uvm/hier/usr/share/untangle/bin/ut-upgrade.py

```diff
@@ -157,6 +176,8 @@ def check_dpkg():
 log_date("")
 log("")
 
+install_frr()
```

Comment:
NO.

Frr install should be handled entirely by debian depdendencies.  Besides, since this file isn't installed on an upgrade, it won't work anyway.